src/main/java/com/answerdigital/benhession/academy/answerkingweek2/model/OrderItem.java

```diff
@@ -13,9 +18,15 @@
 import javax.persistence.Table;
 import javax.persistence.Transient;
 import java.math.BigDecimal;
+import java.util.Objects;
 
 @Entity
 @Table(name = "order_item")
+@NoArgsConstructor
```

Comment:
Another one, can we move the annotations so @NoArgsConstructor and @AllArgsConstructor are next to each other